index.js

```diff
@@ -1,18 +1,44 @@
 /***** Deliverable 1 *****/
-const header = document.querySelector("#header")
+const header = document.querySelector("h1#header")
 console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
 
-
+header.style.color = 'yellow'
+header.style.background = 'blue'
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+
+// name
+let travelerName = document.querySelector("h2")
+travelerName.textContent = `${traveler.name}`
+
+// nickname
+let travelerNickname = document.querySelector("em")
+travelerNickname.textContent = `${traveler.nickname}`
```

Comment:
String interpolation is not necessary here - you should only use string interpolation when you're creating a string our of a variable **and** some additional content, like this:

```js
travelerName.textContent = `My name is ${traveler.name}`
```

For the code above, this will work just fine without interpolation:

```js
let travelerName = document.querySelector("h2")
travelerName.textContent = traveler.name

let travelerNickname = document.querySelector("em")
travelerNickname.textContent = traveler.nickname
```